### DISCUSSION TITLE
Inverter IP Address change

### DISCUSSION THREAD

**Discussion by GregEdge** [WillCodeForCats/solaredge-modbus-multi#262] [Q&A] [answered]
A remote update of my system caused the Inverter IP address to change. Now the Integration fails in HA as it has the old IP address. How do I update the IP address in the modbus multi Integration as it is not on the configuration card ?
Very new to HA.

thanks

**Comment by WillCodeForCats** [comment, marked as answer]
You will have to remove and re-add the integration to change the IP address. Home Assistant unfortunately does not support updating or editing the integration configuration after it's been added. It is possible to edit the raw config data in the `.storage` directory and change it that way, but this isn't recommend for new users.

Ref: issue #26 and https://github.com/home-assistant/architecture/issues/377 (Ability to edit/update config entries)

**Reply by Daverover66** [reply]
I had the same issue, probably about the same time.  Inverter lost comms and when it reconnected it had been assigned a different IP address by my router.  I was able to see what IP address the modbus integration was looking for so I reserved that address in my router settings and set the inverter to the same address statically. Everything connected and started working again after that.
